ceph-dash-cephadm-e2e-nightly/config/definitions/ceph-dash-cephadm-e2e-nightly.yml

```diff
@@ -0,0 +1,70 @@
+- project:
+    name: ceph-dash-cephadm-e2e-nightly
+    ceph_branch:
+      - master
+      - pacific
+      - octopus
+    jobs:
+      - '{name}-{ceph_branch}'
+
+- job-template:
+    name: '{name}-{ceph_branch}'
+    display-name: '{name}-{ceph_branch}'
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && focal && x86_64
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+      - inject:
+          properties-content: |
+            TERM=xterm
+
+    triggers:
+      - timed: '@midnight'
+
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph.git
+          branches:
+            - '{ceph_branch}'
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          shallow-clone: true
+          wipe-workspace: true
```

Comment:
Instead of https://github.com/ceph/ceph-build/pull/1936/commits/5a209bedfb3ea6edc6325c7ed6fb4fbce1065ebd, my suggestion would be to add `basedir: ceph` here.  Then the job workspace will look like

`/home/jenkins-build/build/workspace/ceph-dashboard-cephadm-e2e-nightly-master/ceph` and you can call your scripts from there.  Then the [sharedfolders](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/ci/cephadm/ceph_cluster.yml#L29) name will just be `ceph`.

Reply:
Thanks @djgalloway Updated the PR. let me know of what you think